.github/workflows/tox-pytest.yml

```diff
@@ -34,6 +34,15 @@ jobs:
           conda config --show
           printenv | sort
 
+      - name: Set default gcp credentials
+        id: gcloud-auth
+        continue-on-error: true
+        uses: "google-github-actions/auth@v1"
+        with:
+          # you'll need to update the WIF setup in pudl/terraform to allow actions from this repo to access the tox-pytest service account!
```

Comment:
Does this note pertain to cheshire? Or do you mean when someone creates a new repo from this template repo?

Reply:
This is for "when someone creates a new repo from this template repo" - each one of our WIF associations is currently restricted to one repo.